react/players/VimeoPlayer.tsx

```diff
@@ -15,8 +16,9 @@ const VimeoPlayer: FunctionComponent<VideoPlayer> = ({
   loop,
   src,
   description,
+  classes,
 }) => {
-  const handles = useCssHandles(CSS_HANDLES)
+  const { handles } = useCssHandles(CSS_HANDLES, { classes })
```

Comment:
I think the types of the `react/Video.tsx` which is the component the user can import is going to be wrong

Reply:
We can try to fix this in another PR and just let this one be for the dangerousSetInnerHtml